packages/cms-cli/commands/create.js

```diff
@@ -27,6 +27,7 @@ const TYPES = {
   'global-partial': 'global-partial',
   module: 'module',
   template: 'template',
+  email: 'email',
```

Comment:
Not sure that "email" is the right term here since it is creating an email template not an email. I think we should use `email-template` or add a prompt flow.

Reply:
I updated this and went down the prompt flow path.